src/webui/www/private/scripts/mocha-init.js

```diff
@@ -979,18 +979,34 @@ const initializeWindows = function() {
         }
     };
 
-    ['pause', 'resume'].each(function(item) {
-        addClickEvent(item + 'All', function(e) {
-            new Event(e).stop();
+    addClickEvent('pauseAll', (e) => {
+		new Event(e).stop();
```

Comment:
Seems you accidentally used tabs instead of spaces. That is why CI failed.

Reply:
Sorry I used VS Code and good question why it decided to use tabs now. I had to add a custom keybind now to even "render  whitespaces" ugh. But at least I won't make this mistake again.